action/ons/update.go

```diff
@@ -165,7 +165,7 @@ func runUpdate(ctx *action.Context, tx action.RawTx) (bool, action.Response) {
 			}
 		}
 		d.URI = update.Uri
-	} else if strings.Compare(update.Uri, "") == 0 {
+	} else if ctx.State.Version() >= ctx.Config.ReleaseChanges.DomainChange1 && strings.Compare(update.Uri, "") == 0 {
```

Comment:
Are we checking for the condition that "ctx.Config.ReleaseChanges.DomainChange1" doesn't exist in config.toml?

Reply:
no, this ReleaseChanges.DomainChange1 is defined in a function, and config.toml is just a place to export those configs